spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -198,7 +206,7 @@ def configure_action_cable_to_use_redis
   end
 
   def install_universal_viewer
-    raise '`yarn install` failed!' unless system('./bin/yarn install')
+    raise '`yarn install` failed!' unless system('yarn install')
```

Comment:
`./bin/yarn install` fails while `yarn install` succeeds, which is why I made the switch. Unless there is a particular reason behind the use of `./bin/yarn install`, this change should be ok.